spec/src/main/asciidoc/required-apis.asciidoc

```diff
@@ -46,83 +46,79 @@ MicroProfile applications that make use of APIs or versions not mentioned below
 === Java SE 8
 
 Libraries in use by the MicroProfile leverage new features provided in Java SE 8, therefore Java SE 7 and before are unsupported.
-MicroProfile 3.x implementations may provide support for Java SE 9 and beyond, however Java SE 8 compatibility must be maintained.
+MicroProfile 4.x implementations may provide support for Java SE 9 and beyond, however Java SE 8 compatibility must be maintained.
 
-[[javaee-cdi]]
-=== CDI 2.0
+[[jakarta-cdi]]
+=== Jakarta Contexts and Dependency Injection (CDI) 2.0
 
-CDI provides the base for a growing number of APIs included in MicroProfile 2.0.
-Use of implementations beyond CDI 2.0 in MicroProfile is allowed, however, not required as of version 2.0 of MicroProfile.
+CDI provides the base for a growing number of APIs included in MicroProfile 4.0.
+Use of implementations beyond CDI 2.0 in MicroProfile is allowed, however, not required.
 
+ - https://jakarta.ee/specifications/cdi/2.0/cdi-spec-2.0.html
  - http://cdi-spec.org/
```

Comment:
This should be replaced with https://github.com/eclipse-ee4j/cdi/ (the proper location for the Jakarta CDI spec).

Reply:
Will fix.